### PR TITLE
Update syscall tables and linux headers

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Comment by yuwata** [comment]
CentOS CI (Arch Linux) failure is caused by https://github.com/libcamera-org/libcamera/issues/15.

File: src/basic/syscall-list.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ fsopen
 fspick
 fstat
 fstat64
+fstatat
```
Comment (YHNdnzj) [inline src/basic/syscall-list.txt:98]: Hmm? `fstatat` isn't a syscall on any architecture?

Comment (yuwata) [reply src/basic/syscall-list.txt:98]: I am not sure. I just ran `ninja update-syscall-tables`.

Comment (YHNdnzj) [reply src/basic/syscall-list.txt:98]: Maybe also add it to the `file-system` seccomp set then?

Comment (yuwata) [reply src/basic/syscall-list.txt:98]: > Maybe also add it to the `file-system` seccomp set then?

Done.

**Review by bluca — APPROVED** [review]

**Comment by kbingham** [comment]
> CentOS CI (Arch Linux) failure is caused by [libcamera-org/libcamera#15](https://github.com/libcamera-org/libcamera/issues/15).

@yuwata Is it feasible for you to test this patch https://patchwork.libcamera.org/patch/20820/ which will report the errno / failure that occured here please?

**Comment by yuwata** [comment]
@kbingham 
Sorry, the failure happens only on CIs, and I do not have enough time to test the patch.
I hope it will be tested by our CIs after the next release of libcamera is released.

**Pull request merged**